astropy/modeling/tests/test_models.py

```diff
@@ -752,6 +752,16 @@ def test_tabular1d_inverse():
     assert_allclose(t(result), 100)
 
 
+@pytest.mark.skipif("not HAS_SCIPY_14")
```

Comment:
`HAS_SCIPY_14` can simply be `HAS_SCIPY` now. Somehow missed this one in #10345 

https://github.com/astropy/astropy/blob/ee8de6ba65e2d265dcb1d6ee48d658f3564fc99e/setup.cfg#L61

Reply:
Ok, thanks, I'll update all of these in this file